app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/base/ActivityView.java

```diff
@@ -0,0 +1,19 @@
+package com.mobile.tandil.javabaseproject.mvp.view.base;
+
+import android.app.Activity;
+import androidx.annotation.Nullable;
+import java.lang.ref.WeakReference;
+
+public class ActivityView {
+
+    private WeakReference<Activity> activityRef;
+
+    public ActivityView(Activity activity) {
+        activityRef = new WeakReference<>(activity);
+    }
+
+    @Nullable
+    public Activity getActivity() {
+        return activityRef.get();
+    }
+}
```

Comment:
add an empty line at the end of the file to avoid this error

Reply:
solved